packages/salesforcedx-utils-vscode/test/unit/helpers/utils.test.ts

```diff
@@ -8,11 +8,7 @@
 import { expect } from 'chai';
 import * as fs from 'fs';
 import * as sinon from 'sinon';
-import {
-  extractJsonObject,
-  flushFilePath,
-  flushFilePaths
-} from '../../../src/helpers';
+import { extractJsonObject, fileUtils } from '../../../src/helpers';
```

Comment:
This is an example where grouping all the exports from vscode-utils resulted in stubs not working as expected.  It's due to loading the individually export function and since when we hit the first import from vscode-utils all modules are loaded we have a value vs reference stubbing issue.  I worked around this by grouping function in the export.

Reply:
Not able to see the same changes in `gbockus/allow-utils-in-faux` branch. have pulled the latest.

Reply:
`gbockus/allow-utils-in-faux-4`?